src/pages/Home/Home.tsx

```diff
@@ -67,6 +78,20 @@ export const Home = () => {
           <div />
         </div>
       </section>
+      <section className={style.imageSection}>
```

Comment:
Let's continue the background of what was above with the dogs.
![image](https://github.com/Kenzie-Tech-Solutions/theRoyalCanine/assets/88916004/9a539506-e399-4185-ab08-57ad0c6560d7)

Lets slide those images to the middle and make them space evenly. 
In regards to that picture of the client, we might just need to crop the image ourselves and test it out so it does not look as distorted like we see it. Make the images smaller. Let's go with something like 450 L and W and obviously responsive.